e2e/_suites/ingest-manager/features/stand_alone_agent.feature

```diff
@@ -5,18 +5,30 @@ Feature: Stand-alone Agent
   with them
 
 @start-agent
-Scenario: Starting the agent starts backend processes
-  When a stand-alone agent is deployed
+Scenario Outline: Starting the <image> agent starts backend processes
+  When a "<image>" stand-alone agent is deployed
```

Comment:
I find confusing the use of "stand-alone" here. Does it refer to deploying the agent using a docker image? If that is the case, could we specifically say that?
This way it would be more clear that this feature tests the agent deployed with a docker image, while other features test the agent installed using some package.

```suggestion
Scenario Outline: Starting the <image> docker image of the agent starts backend processes
  When an agent is deployed using the "<image>" docker image
```

Then we could rename the feature to `docker_agent.feature` or something like this. If we don't want to couple the feature to specific technology we can use "container" instead of "docker".

Reply:
@EricDavisX wdyt?

Reply:
I appreciate and understand the idea.  To me, using Docker is just a means to test the 'Stand-alone' Agent mode which is a feature of Elastic Agent, as opposed to 'Docker' or 'Container' which is just how we've implemented the test.  I think I'd like to keep it as is, if it were up to me - it keeps the test scenarios feature oriented instead of technology aligned.

Further, it kinda highlights that we could implement other support for 'stand-alone' mode in that we could start with a blank Centos (or Debian or Red Hat ubi) and deploy the Agent as part of the test.  Doing that would require picking apart the current example / default config of the agent as it is packaged and replacing parts of it and so is a bit more work than what we have already done here which meets the needs of a basic test (and I admit we kind of stumbled into this usage on accident, using Docker that is, but it fits fine).  

Finally, the pure 'stand-alone' mode is not a focus now, so I am content to leave it as is and not spend time on reimplementing or changing test support - the real goal is to move faster to build in more OS support (in any fashion) and more features etc.  So, we're doing great!

Reply:
Ok, I get it, so it is just a feature that happens to be tested with the docker image, but it could be tested with other deployment approaches.